### PR TITLE
Team5 signoff user interface npatel45

### PR DESCRIPTION
**Pull request by nidhaypatel**

### CODE REVIEW

File: Documentation/Signoffs/User-Interface-npatel45.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Vending Machine Subsystem - User-Interface
+### Function of the Subsystem:
+---
+When designing vending machine the main component of the machine is to get input from the user and process that information and send that information to the rest of the parts corresponding to it. So, in the same way for ECE board vending machine the student must be able to input their information, and select products from given set of  choices so to target this requirement, UI (User Interface) subsystem is required.
+  
+•	Receive information from student using medium of a touch screen LCD (Liquid Crystal Display)
+ 
+### Specifications & Constraints:
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/User-Interface-npatel45.md:6]: These constraints seem to not cover things that are important and instead say things that shouldn't need to be said (recognize depression of all keys?)

Comment (JesseTNRoberts) [reply Documentation/Signoffs/User-Interface-npatel45.md:6]: Without a clear understanding of the system's constraints how can it be designed and analyzed?

File: Documentation/Signoffs/User-Interface-npatel45.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# Vending Machine Subsystem - User-Interface
+### Function of the Subsystem:
+---
+When designing vending machine the main component of the machine is to get input from the user and process that information and send that information to the rest of the parts corresponding to it. So, in the same way for ECE board vending machine the student must be able to input their information, and select products from given set of  choices so to target this requirement, UI (User Interface) subsystem is required.
+ 
+### Specifications & Constraints:
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/User-Interface-npatel45.md:6]: What about size? Will this display actually allow the operator and users to read what is needed? Is there a good reason to use a very small display? Big displays give the impression of quality.

File: Documentation/Signoffs/User-Interface-npatel45.md [hunk 3]
```diff
@@ -9,7 +9,8 @@ When designing vending machine the main component of the machine is to get input
   1. UI must not operate at greater than 5 V 0.5 A because of mini-PC output rating [1].
   2. UI must run Windows/Linux environment to run Kiosk application.
   3. Must have HDMI or type C or USB port to connect to mini-PC.
- 
+  4. UI must be at least 7" (diagonal) for readability.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/User-Interface-npatel45.md]: this comes from where? Arbitrary.

This in no way addresses the previous feedback.

**Review by JesseTNRoberts — COMMENTED** [review]
The constraints look much better! Do you think that they should include anything about ease of theft?

**Comment by nidhaypatel** [comment]
> The constraints look much better! Do you think that they should include anything about ease of theft?

The machine will be in Eastman lounge so students and faculty would only have access to it, also the LCD that the team is thinking to go with is a portable monitor so there is no way to secure it unless we screw the soft cover that comes with LCD.

**Review by JesseTNRoberts — COMMENTED** [review]
Exactly, it will be open to the public. Should you use a display that can't be anchored? (Obviously I don't think so)

**Comment by nidhaypatel** [comment]
> Exactly, it will be open to the public. Should you use a display that can't be anchored? (Obviously I don't think so)

No, it is going to be portable touch screen monitor so the only way I can think of is to change the location of machine and put it in ECE office.

**Review by JesseTNRoberts — APPROVED** [review]
Remember, we will not order this. We will instead use in department resources.

**Pull request merged**